Algorithm.FSharp/Notes.md

```diff
@@ -0,0 +1,25 @@
+﻿## F# Notes
```

Comment:
Great notes! Please rename file to "readme.md" so it'll be displayed on navigation to the folder.

Reply:
done